cmd/snap/notes.go

```diff
@@ -52,6 +52,7 @@ func formatPrice(val float64, currency string) string {
 // snap, in order to present a brief summary of it.
 type Notes struct {
 	Price            string
+	Health           string
```

Comment:
shouldn't it come later in the struct, given it is used last?

Reply:
ah,  I would move

```
    Health string
    Price string
```

to the bottom of the struct in that order

Reply:
and SnapType?

Reply:
can stay first field